pkg/apis/pipelinesascode/v1alpha1/types.go

```diff
@@ -169,6 +171,15 @@ type GitlabSettings struct {
 	CommentStrategy string `json:"comment_strategy,omitempty"`
 }
 
+type GithubSettings struct {
+	// CommentStrategy defines how GitLab comments are handled for pipeline results.
+	// Options:
+	// - 'disable_all': Disables all comments on merge requests
+	// +optional
+	// +kubebuilder:validation:Enum="";update_existing;disable_all
```

Comment:
```suggestion
	// +kubebuilder:validation:Enum="";disable_all
```

if you don't wanna keep it at this moment